### PR TITLE
"get_file_report" method can be used "allinfo" option when using Private API

### PR DESCRIPTION
**Pull request by kobayanes**
I wanted a more information in the report, so I added "allinfo" argument it.

This is my first pull request.
If I wrong something, I'll apologize it.

### CODE REVIEW

File: virus_total_apis/api.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def rescan_file(self, this_hash, timeout=None):
 
         return _return_response_and_status_code(response)
 
-    def get_file_report(self, this_hash, timeout=None):
+    def get_file_report(self, this_hash, timeout=None, allinfo=None):
```
Comment (KadenLNelson) [inline virus_total_apis/api.py]: You've mentioned in your description that your changes were intended for the VT private api but you have made changes to the public api definition in `api.py`

**Comment by KadenLNelson** [comment]
https://github.com/blacktop/virustotal-api/blob/4e01e1c6d87255ec8370ac2a4ee16edce00e1e86/virus_total_apis/api.py#L382

On this line in the private api we already have a predefined integer value of 1 for the `allinfo` parameter for the method `get_file_report`. I think we should default this value to false, what about you @blacktop?

**Comment by blacktop** [comment]
@kobayanes you can see in the [Public API docs](https://www.virustotal.com/en/documentation/public-api/) that **allinfo** is not a supported (or mentioned) parameter.  They want you to pay a LOT of money to get all the info 😉 

@KadenLNelson I believe if you paid all that 💰💰💰you would always want all the info, and if you didn't that would be more of a "edge case" which is why it defaults to true (1).

**Comment by blacktop** [comment]
Thank you @KadenLNelson for taking a look at this. 👍

**Comment by kobayanes** [comment]
@KadenLNelson @blacktop 
I'm very sorry ... I was wrong.
It is Public API rather than Private API. I understood.
Thank you for your conversation and pointing out. I'll read codes more.

**Pull request closed without merging**